packages/terra-switch/src/terra-dev-site/doc/example/DisabledSwitch.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Switch from 'terra-switch';
+
+export default () => (
+  <div>
+    <Switch
+      id="disabledSwitch"
+      labelId="label2"
+      checked
+      label="Label"
+      disabled
+    />
+  </div>
```

Comment:
The divs in each of these examples can be removed

Reply:
[removed](https://github.com/cerner/terra-core/pull/2846/commits/a30ab9bf52d60ad2b5b83dbe86defbb2a580b981#diff-f53740e55f04bd91c91500b286c5184c)